test/dotenv-flow.spec.js

```diff
@@ -264,6 +264,39 @@ describe('dotenv-flow', () => {
     });
   });
 
+  describe('when the `node_env` option is provided', () => {
```

Comment:
I suggest you to write a test when you provide different environments through `node_env`, and they all have a priority over defined `process.env.NODE_ENV`.

Reply:
I actually Already did this. But I used the `.env.test*` files and the string 'test' as my field of choice. So I'll change that up to use a proper choice thats not excluded